build/ci/production-values.yaml

```diff
@@ -34,6 +34,11 @@ web:
       ref: logging-secret-production
     schoolLookup:
       ref: school-lookup-data-production
+linker:
+  enabled: true
+  containerImage:
```

Comment:
I want to make sure the image backing the linker pod will always be the latest image available. I copied this syntax from the `web` block.

Reply:
Ah - well spotted.  You will also need to extend `build/ci/production-helm-deploy.sh` to also inject the web image into this field

```sh
yq e -i '.web.containerImage.imageRegistry = strenv(WEB_IMAGE)' $1
yq e -i '.nodejs.containerImage.imageRegistry = strenv(NODE_IMAGE)' $1
yq e -i '.nginx.containerImage.imageRegistry = strenv(ASSET_IMAGE)' $1
yq e -i '.monitor.containerImage.imageRegistry = strenv(WEB_IMAGE)' $1
yq e -i '.web.containerImage.tag = strenv(TAG)' $1
yq e -i '.nodejs.containerImage.tag = strenv(TAG)' $1
yq e -i '.nginx.containerImage.tag = strenv(TAG)' $1
yq e -i '.monitor.containerImage.tag = strenv(TAG)' $1
```